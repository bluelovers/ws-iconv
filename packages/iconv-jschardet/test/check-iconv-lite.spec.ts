import iconvLite from 'iconv-lite';

test(`iconvLite`, () =>
{

	expect(iconvLite).toMatchSnapshot();

});

test(`getCodec`, () =>
{

	let actual;
	let expected;

	expect(iconvLite).toHaveProperty('getCodec');

	// @ts-ignore
	expect(typeof iconvLite.getCodec).toStrictEqual('function');

});
