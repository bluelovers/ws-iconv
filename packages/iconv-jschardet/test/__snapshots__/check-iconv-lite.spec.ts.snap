// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`iconvLite 1`] = `
Object {
  "IconvLiteDecoderStream": [Function],
  "IconvLiteEncoderStream": [Function],
  "_canonicalizeEncoding": [Function],
  "_codecDataCache": Object {},
  "decode": [Function],
  "decodeStream": [Function],
  "defaultCharSingleByte": "?",
  "defaultCharUnicode": "ï¿½",
  "enableStreamingAPI": [Function],
  "encode": [Function],
  "encodeStream": [Function],
  "encodingExists": [Function],
  "encodings": null,
  "fromEncoding": [Function],
  "getCodec": [Function],
  "getDecoder": [Function],
  "getEncoder": [Function],
  "supportsStreams": true,
  "toEncoding": [Function],
}
`;
